gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +446,67 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effectiveareatable(self, offset, energ_lo = None, energ_hi = None):
+        """Evaluate at a given offset and return effective area table object.
+
+        The energy thresholds in the effective area table object are not set.
+        If the effective area table is intended to be used for spectral analysis,
+        the final energy binning should be given at this point, since the
+        effective area table class is not able to perform interpolation at
+        this point.
+
+        Parameters
+        ----------
+        offset : `~astropy.coordinates.Angle`
+            offset
+        energy : `~astropy.units.Quantity`
+            energy
+
+        Returns
+        -------
+        eff_area_table : `EffectiveAreaTable`
+             Effective area table class. 
+        """
+
+        if energ_lo is None and energ_lo is None:
+            energ_lo = self.energ_lo  
+            energ_hi = self.energ_hi
+        elif energ_lo is None or energ_hi is None:
+            raise ValueError("Only 1 energy vector given, need 2")
+        if not isinstance(offset, Angle):
+            raise ValueError("Offset must be an angle object.")
+        if not isinstance(energ_lo, Quantity) or not isinstance(energ_hi, Quantity):
+            raise ValueError("Energy must be a Quantity object.")
+        if len(energ_lo) != len(energ_hi):
+            raise ValueError("Energy Vectors must have same dimension")
+        
+        energy = np.sqrt(energ_lo * energ_hi)
+        area = self.evaluate(offset,energy) 
+
+        return EffectiveAreaTable(energ_lo, energ_hi, area)
+
+    def write_ARF(self, filename, offset, energ_lo = None, energ_hi = None):
```

Comment:
Do you think it's worth adding this 2-line convenience method?
I'm leaning very slightly towards no, but if you like it, keep it!

With it you'd write
```python
effective_area_2d.write_arf('arf.fits', Angle(2, 'deg'))
```
Without it
```python
effective_area_2d.to_effective_area(Angle(2, 'deg')).write('arf.fits', format='arf'))
```